### PR TITLE
Adding new Managed VMs samples.

### PR DESCRIPTION
**Pull request by jonparrott**
Existing samples have been renamed:
* django_tutorial -> django_cloudsql
* hello_world -> hello_world_compat
* hello_world_custom -> (superseded by) hello_world

### CODE REVIEW

**Comment by jonparrott** [comment]
Confirmed - No docs reference any renamed folders.

**Comment by jonparrott** [comment]
The tests currently do not run - we'll need to configure our project and add all of the necessary config stuff in our secrets, do you mind saving that for another PR, @waprin?

**Comment by waprin** [comment]
lgtm

**Comment by waprin** [comment]
Im confused about the testing comment, it looks like travis ran?

**Comment by jonparrott** [comment]
> Im confused about the testing comment, it looks like travis ran?

Travis ran, but the new tests in this folder did not get picked up. A `__init__.py` file needs to be placed in this folder for that to work (and this folder needs to be added to `.coveragerc`).

File: managed_vms/cloudsql/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Before you can run or deploy the sample, you will need to do the following:
+
+1. Create a Cloud SQL instance. You can do this from the [Google Developers Console](https://console.developers.google.com) or via the [Cloud SDK](https://cloud.google.com/sdk). To create it via the SDK use the following command:
+
+        $ gcloud sql instances create [your-instance-name] \
+            --assign-ip \
+            --authorized-networks 0.0.0.0/0 \
+            --tier D0
+
+2. Create a new user and database for the application. The easiest way to do this is via the [Google Developers Console](https://console.developers.google.com/project/_/sql/instances/example-instance2/access-control/users). Alternatively, you can use MySQL tools such as the command line client or workbench.
```
Comment (waprin) [inline managed_vms/cloudsql/README.md]: If you want to recommend using the MySQL client you have to add the `gcloud sql instances set-root-password` command to give them an initial login.

Comment (jonparrott) [inline managed_vms/cloudsql/README.md]: Fixed.

**Pull request merged**